Modules/FlexForm.general.js

```diff
@@ -1202,13 +1202,26 @@ function createAlertsIfNeeded () {
 	}
 }
 
+function ffOnSelect2OpenedFocus() {
+	$(document).on('select2:open', () => {
```

Comment:
Please put comment here that links to this:
https://github.com/select2/select2/issues/5993#issuecomment-1050841204